### PR TITLE
adapt MapSequence example and new folder examples/finding_data/

### PR DESCRIPTION
**Pull request by STBadman**
Added files examples/finding_data/adapt_helpers.py and examples/adapt_MapSequence.py which respectively help download a sample adapt .fts file, and present as an example the parsing of this fts file to a `sunpy.map.MapSequence` object

### CODE REVIEW

**Comment by STBadman** [comment]
Relates to Issue #166

**Comment by dstansby** [comment]
Hmm I wonder where has the doc build gone

File: examples/finding_data/adapt_MapSequence.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"""
+Pares ADAPT Ensemble
+====================
```
Comment (dstansby) [inline examples/finding_data/adapt_MapSequence.py:3]:
```suggestion
Parsing ADAPT Ensemble .fits files
==================================
```

File: examples/finding_data/adapt_MapSequence.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+* Parse an adapt FITS file as a `sunpy.map.MapSequence`.
```
Comment (dstansby) [inline examples/finding_data/adapt_MapSequence.py:5]:
```suggestion
Parse an ADAPT FITS file into a `sunpy.map.MapSequence`. 
```

**Pull request merged**